ci/infra/jobs/build_amis/dw-al2-concourse-ami.yml

```diff
@@ -0,0 +1,92 @@
+resources:
+  - name: dw-al2-concourse-ami
+    .: (( inject meta.resources.ami-builder-configs ))
+    source:
+      paths:
+        - dw-al2-concourse-ami/*
+        - generic_packer_template.json.j2
+
+  - name: dw-al2-concourse-ami-pr
+    .: (( inject meta.resources.ami-builder-configs-pr ))
+    source:
+      paths:
+        - dw-al2-concourse-ami/*
+        - generic_packer_template.json.j2
+
+jobs:
+  - name: dw-al2-concourse-ami
+    plan:
+      - get: dw-al2-concourse-ami
+        trigger: true
+      - .: (( inject meta.plan.get-al2-hardened-ami ))
+      - .: (( inject meta.plan.generate-manifest ))
+        input_mapping:
+          source_ami: dw-al2-hardened-ami
+        config:
+          inputs:
+            - name: source_ami
+          params:
+            AMI_NAME: dw-al2-concourse-ami
+            SOURCE_AMI_NAME: dw-al2-hardened-ami-*
+            SOURCE_AMI_OWNER: ((aws_account.management))
+            PROVISION_SCRIPT_KEYS: '["dw-al2-concourse-ami/dw-al2-concourse-ami/dw-concourse-ami-install.sh"]'
+            PROVISIONER_TYPE_FILE_SOURCE: "dw-al2-concourse-ami/dw-al2-concourse-ami/"
+            CONCOURSE_VERSION: 6.4.1
+      - .: (( inject meta.plan.packer-bootstrap ))
+        config:
+          run:
+            dir: dw-al2-concourse-ami
+          inputs:
+            - name: dw-al2-concourse-ami
+      - .: (( inject meta.plan.build-ami ))
+        config:
+          inputs:
+            - name: dw-al2-concourse-ami
+
+  - name: dw-al2-concourse-ami-pr
+    max_in_flight: 1
+    plan:
+      - get: dw-al2-concourse-ami-pr
+        trigger: true
+        version: every
+      - put: dw-al2-concourse-ami-pr
+        params:
+          path: dw-al2-concourse-ami-pr
+          status: pending
+      - .: (( inject meta.plan.get-al2-hardened-ami ))
+      - .: (( inject meta.plan.generate-manifest ))
+        input_mapping:
+          source_ami: dw-al2-hardened-ami
+        config:
+          inputs:
+            - name: source_ami
```

Comment:
this should go when you move it to meta (as per previous comment). I won't add any further PR comments about it

Reply:
Done.